.github/workflows/docker_management.test-PR.yml

```diff
@@ -0,0 +1,82 @@
+name: Build and test image when PR is raised
```

Comment:
as this is pull request workflow, let's remove "when PR is raised".

What image is it? Let's name this "Build docker image"